### ISSUE TITLE
Community Call #37: Design-a-thon planning

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Meta#327]
```meta
Date: 2022-09-07
Time: 17:00 
Timezone: UTC
Duration: 1h
UTCTime: 2022-09-07 17:00 UTC
```

![bitcoin-design-community-call-37](https://user-images.githubusercontent.com/695901/187638053-f1c9b2f9-b1d7-4da3-94ab-eaad31de1e05.jpg)

### Agenda

To be determined as we get closer to the date. Feel free to post a topic to discuss in the comments below.

### Check your timezone

https://everytimezone.com/s/f023e231

### Join the call

The call will be hosted via Twitter Spaces.

### Calendar

Subscribe to the [Bitcoin Design calendar](https://bitcoin.design/calendar/).

**Comment by decypherpunk** [comment]
Can't find the Twitter space for this call...help

**Comment by GBKS** [comment]
Argh, I am so sorry, I forgot to update this issue. We had the call on Jitsi this time for a more private group discussion. We discussed planning a designathon event for the community where we set aside a week in October for open and collaborative design exploration. We went over topics of naming, schedule, format, prizes, project ideas, and more. If you want to catch up, the recording is up on [BitcoinTV](https://bitcointv.com/w/qU7GXfDMbj4GFQi2oRDP4s) and [YouTube](https://youtu.be/Ua1v17zoeIA). We are also chatting in the [#playtime](https://bitcoindesign.slack.com/archives/C0402AVBLPQ) channel in Slack. Let me know if you'd like more info. Sorry again.
